content/src/outline.rs

```diff
@@ -293,6 +294,75 @@ impl Contour {
         contour
     }
 
+    #[inline]
+    pub fn from_rect_rounded(rect: RectF, radius: Vector2F) -> Contour {
+        use std::f32::consts::SQRT_2;
+        const QUARTER_ARC_CP_FROM_OUTSIDE: f32 = (3.0 - 4.0 * (SQRT_2 - 1.0)) / 3.0;
```

Comment:
Where does this number come from?

Reply:
It is from https://www.chartwellyorke.com/mx/explore/geometric-modeling/circle-spline-approximation.pdf
page 4. Except that I am using 1 - 4/3 (√2-1) = 3/3 - 4/3 (√2-1) = (3 - 4(√2-1))/3